Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/HandlerTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Backbone.Modules.Quotas.Application.IntegrationEvents.Incoming;
+using Backbone.Modules.Quotas.Domain.Aggregates.Entities;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.Identities
+{
+    public class HandlerTests
+    {
+        private readonly ILogger<IdentityCreatedIntegrationEventHandler> _logger;
+
+        public HandlerTests() { }
+
+        [Fact]
+        public async void Returns_add_when_identities_created()
```

Comment:
Rename this test to 'Successfully_creates_identity_after_consuming_integration_event'